profiler/src/ProfilerEngine/Datadog.Profiler.Native.Linux/OsSpecificApi.cpp

```diff
@@ -60,10 +69,10 @@ bool GetCpuInfo(pid_t tid, bool& isRunning, uint64_t& cpuTime)
     char state = ' ';   // 3rd position  and 'R' for Running
     int userTime = 0;   // 14th position in clock ticks
     int kernelTime = 0; // 15th position in clock ticks
-    bool success =
-        fscanf(file, "%*s %*s %c %*s %*s %*s %*s %*s %*s %*s %*s %*s %*s %d %d",
-               &state, &userTime, &kernelTime) == 3;
-    fclose(file);
+
+    auto pos = sline.find_last_of(")");
```

Comment:
for the question (I asked it myself): are parenthesis always used ?
answerd: This is the code in the kernel that write in that file and there will always be parenthesis https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/fs/proc/array.c#n567